src/components/checkbox-drawer/checkbox-drawer.js

```diff
@@ -64,17 +53,14 @@ class CheckboxDrawer extends LocalizeLabsElement(SkeletonMixin(LitElement)) {
 					class="d2l-input-checkbox"
 					?skeleton="${this.skeleton}"
 					description=${this.localize(`components:checkboxDrawer:checkbox${this.checked ? 'Expanded' : 'Collapsed'}`)}
-					@change="${this._onCheckboxChange}">${this.label}</d2l-input-checkbox>
+					@change="${this._onCheckboxChange}">
+					${this.label}
+					<div slot="inline-help">${this.description}</div>
+					<d2l-expand-collapse-content slot="supporting" ?expanded="${this.checked}" aria-busy=${this._expandCollapseBusy ? 'true' : 'false'} @d2l-expand-collapse-content-expand="${this._onExpandCollapseContentToggled}" @d2l-expand-collapse-content-collapse="${this._onExpandCollapseContentToggled}">
+						<slot></slot>
+					</d2l-expand-collapse-content>
+				</d2l-input-checkbox>
 			`}
-			<d2l-input-checkbox-spacer class="d2l-input-checkbox-spacer">
-				<div class="d2l-input-checkbox-description d2l-skeletize">${this.description}</div>
-			</d2l-input-checkbox-spacer>
-			<d2l-input-checkbox-spacer class="d2l-input-checkbox-spacer">
```

Comment:
The rest of this can go in the `supporting` slot.

<img width="699" alt="Screenshot 2025-04-10 at 4 14 51 PM" src="https://github.com/user-attachments/assets/22623402-034b-46d8-9ae9-a6c478fa4e25" />